GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -202,11 +204,24 @@ private void OnGuiCreateBuildSelect()
             EditorGUILayout.Space();
             EditorGUILayout.BeginHorizontal();
             EditorGUILayout.LabelField("APK File Name", GUILayout.MinWidth(FieldMinWidth));
-            QuickDeployConfig.Config.apkFileName = EditorGUILayout.TextField(QuickDeployConfig.Config.apkFileName,
-                GUILayout.MinWidth(FieldMinWidth));
+            QuickDeployConfig.Config.apkFileName =
+                EditorGUILayout.TextField(QuickDeployConfig.Config.apkFileName, GUILayout.MinWidth(FieldMinWidth));
+            if (GUILayout.Button("Browse", GUILayout.Width(ShortButtonWidth)))
+            {
+                const string defaultApkFileName = "base.apk";
+                QuickDeployConfig.Config.apkFileName = EditorUtility.SaveFilePanel("Choose file name and location",
+                    Path.GetDirectoryName(defaultApkFileName),
+                    Path.GetFileName(defaultApkFileName),
```

Comment:
Is this always "base.apk"? Could just put that directly.

Reply:
Read comment above this